snippets/xaml/wpf/introduction-to-wpf/introduction-to-wpf_12.xaml

```diff
@@ -0,0 +1 @@
+</Window>
```

Comment:
This file is exactly the same as `introduction-to-wpf_5.xaml`, `introduction-to-wpf_15.xaml`, `introduction-to-wpf_21.xaml`, and `introduction-to-wpf_25.xaml`.